.gitignore

```diff
@@ -13,6 +13,10 @@ tabbycat/static/jsi18n/
 tabbycat/static/vue/
 tabbycat/static/fonts/
 
+# IDE files
+.idea/
+.vscode/
+
```

Comment:
I think these could more efficiently be in your global gitignore, rather than being project-specific.

Reply:
True, will fix that